plugins/shortcuts/src/AddShortcut.tsx

```diff
@@ -59,6 +65,7 @@ export const AddShortcut = ({ onClose, anchorEl, api }: Props) => {
 
     try {
       await api.add(shortcut);
+      analytics.captureEvent('add', url);
```

Comment:
A thought occurred to me: the fact that we expose this as a hook means it's...  Annoying at best (impossible at worst) to use this hook outside the scope of a component.

...But maybe that's actually a desirable property of the API.  It discourages use as a general-purpose logging utility, and makes the happy-path instrumenting user interactions in react component-land.